run-test.sh

```diff
@@ -128,6 +128,12 @@ create_test_overlay()
 	exit 1
   fi
   find $CoreFxBins -name '*.dll' -exec cp '{}' "$OverlayDir" ";"
+
+  # Then the native linux CoreFX binaries
+  #
+  # TODO: Currently, CI does not build the native CoreFX components so build them here
+  #       in the test phase for now.
+  ( $ProjectRoot/src/Native/build.sh && cp $ProjectRoot/bin/$OS.x64.$Configuration/Native/*.so $OverlayDir ) || exit 1
```

Comment:
OK. Can I leave the path handling to you once we have the proper legs? We need to (1) build src/Native on the target OS and (2) get everything in bin/OS.arch.config/Native in to the overlay. I'm not sure if that can be inferred from the existing args or not.